### ISSUE TITLE
Ignoring "vaov" & "vOffset" parameters in JSON tour

### ISSUE THREAD

**Issue by evdelen** [mpetroff/pannellum#69]
I've been creating some real estate tours and have had spheres that aren't displaying properly. It seems that when using a JSON tour Pannellum is ignoring the "vaov" and "vOffset" parameters within a scene. I've tried both using an integer as well as a string and in both cases the value is ignored with the image being displayed in the centre of the scene with identical black spots appearing above and beneath the image.

Example can be seen here: www.evolutionsmc.com/real-estate/204Alton/pannellum.htm?tour=assets/tour.json

Thanks.

Edit: I'm using the 2.1.0 release.

**Comment by mpetroff** [comment]
The parameters are being ignored because your images have Photo Sphere XMP data embedded in them, which is being used instead. Adding `"ignoreGPanoXMP": true` to your config file's `default` section will fix this.

It turns out there were two subtle bugs related to `vOffset`, however, which is why you noticed the issue. The `vOffset` parameter needed to be set twice as large as it should have been, and the `vOffset` derived from the Photo Sphere XMP data was half the size it should have been. These bugs led to the offset being four times too small for images with Photo Sphere XMP data. Both are now fixed in Git.

Since the XMP data overriding configuration settings is not behavior that is generally expected, I also modified Pannellum so configuration settings take precedence.
